mesonbuild/interpreter.py

```diff
@@ -2025,7 +2026,7 @@ def get_cross_property_method(self, args, kwargs):
                     'install_man': {'install_dir', 'install_mode'},
                     'install_subdir': {'exclude_files', 'exclude_directories', 'install_dir', 'install_mode', 'strip_directory'},
                     'jar': build.known_jar_kwargs,
-                    'project': {'version', 'meson_version', 'default_options', 'license', 'subproject_dir'},
+                    'project': {'version', 'meson_version', 'default_options', 'metadata'},
```

Comment:
You can't remove existing kwargs, this needs to be backward compatible.

Reply:
Oops that was an overlook